pyproject.toml

```
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pynvml"
+version = "12.0.0"
+description = "Python utilities for the NVIDIA Management Library"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "BSD-3-Clause" }
+requires-python = ">=3.8"
```

Given Python 3.8 is EOL (see [Status of Python versions page]( https://devguide.python.org/versions/ )), think we can safely set the min to Python 3.9

```suggestion
requires-python = ">=3.9"
```